appeals/web/src/server/lib/mappers/appeal/submappers/complete-date.mapper.js

```diff
@@ -6,9 +6,7 @@ export const mapCompleteDate = ({ appealDetails, currentRoute, userHasUpdateCase
 	textSummaryListItem({
 		id: 'complete-date',
 		text: 'Complete',
-		value:
-			dateISOStringToDisplayDate(appealDetails.appealTimetable?.completeDate) ||
-			'Due date not yet set',
+		value: dateISOStringToDisplayDate(appealDetails.appealTimetable?.completeDate),
```

Comment:
Di you mean to remove this default?

Reply:
yes. Agreed with Rachel we need a review and alignment of all defaults - should be all blank at the moment